app/src/main/java/edu/grinnell/kdic/MainActivity.java

```diff
@@ -1,18 +1,19 @@
 package edu.grinnell.kdic;
-
 import android.content.ComponentName;
 import android.content.Intent;
 import android.content.ServiceConnection;
 import android.content.SharedPreferences;
 import android.os.Bundle;
 import android.os.IBinder;
+import android.support.annotation.NonNull;
```

Comment:
I don't think this import is used anywhere, is it? Before you send a pull request, please organize your imports else we'll have cruft in our code